app/models/person.rb

```diff
@@ -136,7 +136,12 @@ class Person < ApplicationRecord
   def person
     self
   end
-    
+
+  def projects
+    # you can't access these through .projects nested associations until they have been saved
+    work_groups.collect(&:project).uniq | group_memberships.collect { |gm| gm.work_group.project }
+  end
+
```

Comment:
What is the reason behind this change?

Reply:
it's not actually a change, but was there before coming from the project hierarchy stuff, so just got moved over